IPython/utils/traitlets.py

```diff
@@ -1635,14 +1635,14 @@ def validate_elements(self, obj, value):
 class Dict(Instance):
     """An instance of a Python dict."""
 
-    def __init__(self, default_value=None, allow_none=True, **metadata):
+    def __init__(self, default_value={}, allow_none=True, **metadata):
```

Comment:
@ssanderson good point.  That suggests the default in master is wrong and @SylvainCorlay 's suggested default of None is the way to go.